static/sass/components/_hero.scss

```diff
@@ -25,3 +25,54 @@
     }
   }
 }
+
+// Hero Images
+// ===========
+
+
+.hero {
+  max-height: 35vh;
+  overflow: hidden;
+  position: relative;
+
+  @include above('medium-page') {
+    max-height: 45vh;
+  }
+
+  + .title {
```

Comment:
I removed max-width and title is full-width at the moment, which is an issue for short titles on wide screens. We can edit after everything (template-grid and this) is merged if we feel we need to.